pkg/backup/backup_manager.go

```diff
@@ -49,40 +49,48 @@ func NewBackupManagerFromWriter(kubecli kubernetes.Interface, bw writer.Writer,
 	}
 }
 
-// SaveSnap uses backup writer to save etcd snapshot to a specified S3 path.
-func (bm *BackupManager) SaveSnap(s3Path string) error {
-	etcdcli, err := bm.etcdClientWithMaxRevision()
+// SaveSnap uses backup writer to save etcd snapshot to a specified S3 path
+// and returns backup etcd server's version and its kv store revision.
```

Comment:
For the code, it's the opposite:
etcd server's revision and etcd version.

Reply:
good catch!